sonar-db/src/test/java/org/sonar/db/ce/CeActivityDaoTest.java

```diff
@@ -123,6 +123,23 @@ public void test_selectByQuery() {
   }
 
   @Test
+  public void test_selectByQuery_pagination() {
+    insert("TASK_1", REPORT, "PROJECT_1", CeActivityDto.Status.SUCCESS);
+    insert("TASK_2", REPORT, "PROJECT_1", CeActivityDto.Status.FAILED);
+    insert("TASK_3", REPORT, "PROJECT_2", CeActivityDto.Status.SUCCESS);
+    insert("TASK_4", "views", null, CeActivityDto.Status.SUCCESS);
+
+    CeTaskQuery query = new CeTaskQuery().setStatuses(Collections.<String>emptyList());
+    assertThat(underTest.selectByQuery(db.getSession(), query, 0, 1)).extracting("uuid").containsOnly("TASK_4");
```

Comment:
for readability of test, I suggest to extract a method like :
```
List<String> selectPageOfUuids(int offset, int pageSize) {}
```
so that assertions look like:
```
assertThat(selectPageOfUuids(1, 1)).containsExactly("TASK_3");
```